cmake/options.cmake

```diff
@@ -200,6 +200,10 @@ macro(drake_setup_options)
     DEPENDS "HAVE_BOT_CORE_LCMTYPES"
     "robotlocomotion LCM types")
 
+  drake_system_dependency(VTK REQUIRES VTK VERSION 5.8
```

Comment:
Whoops! I recently made the same mistake when using Github's UI.

Let's monitor the following continuous OSX CI to see if it errors out:
https://drake-jenkins.csail.mit.edu/view/Continuous%20Production/job/mac-clang-ninja-continuous-debug/843/